src/EPR.RegulatorService.Facade.API/Validators/ReprocessorExporter/Registrations/UpdateTaskStatusRequestValidator.cs

```diff
@@ -0,0 +1,33 @@
+﻿using EPR.RegulatorService.Facade.API.Constants;
+using EPR.RegulatorService.Facade.API.Validators.ReprocessorExporter.Registrations;
+using EPR.RegulatorService.Facade.Core.Enums;
+using EPR.RegulatorService.Facade.Core.Models.ReprocessorExporter.Registrations;
+using FluentValidation;
+
+namespace EPR.RegulatorService.Facade.API.Validators.ReprocessorExporter.Registrations;
+
+public class UpdateTaskStatusRequestValidator : AbstractValidator<UpdateTaskStatusRequestDto>
+{
+    public UpdateTaskStatusRequestValidator()
+    {
+        // Status: Required
+        RuleFor(x => x.Status)
+            .NotNull()
+            .WithMessage(ValidationMessages.StatusRequired)
+            .Must(status => Enum.IsDefined(typeof(RegistrationTaskStatus), status))
+            .WithMessage(ValidationMessages.StatusRequired);
+
+        // Comments: Max Length 
+        RuleFor(x => x.Comments)
+            .MaximumLength(200)
```

Comment:
```suggestion
            .MaximumLength(500)
```

Requirements changed to accept 500 Characters